### ISSUE TITLE
[captcha] Captcha plugin - reload option not working on login page

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#312]
```
Dear Roland,

First of all, congratulations for the work with the plugins for Roundcube. I 
didn't know this webmail solution until last month, and now I'm testing it and 
some plugins (including some of yours).

Well, my problem is reported as follows. I couldn't find a way to put the 
captcha's reload button to work properly on the Roundcube login page. 

In fact, this button does not even appear on the default configuration. To 
solve this, I had to copy the following line from 
/plugins/captcha/skins/default/templates/captcha.html to 
/skins/default/templates/login.html:

<input onclick="document.location.href = self.location.href" type="button" 
class="button" value="<roundcube:label name='captcha.reload' />" />

The button appears now, but when clicked the whole page is reloaded - not only 
the captcha image. Thus, the user has to type the login credentials again.

Is there a way to solve it?

Below I describe my config:

On /config/main.inc.php

   $rcmail_config['plugins'] = array('captcha',... - and others

On /plugins/captcha/captcha.php:

   public $task = "login|mail|logout|settings"; (not changed)

On /plugins/captcha/config.inc.php:


   $rcmail_config['captcha_auth'] = TRUE;

   $rcmail_config['captcha_page'] = array("contactus.contactus","pwtools.recpw","register.register");

What steps will reproduce the problem?
1. Force the appearing of captcha.reload object on the login.html page;
2. Try to reload the captcha image;

What is the expected output? What do you see instead?
Only the captcha image should be reloaded, but the whole page is, which clears 
the other input fields.

What version of the product are you using? On what operating system?

Roundcube v0.7.1 + Captcha plugin v 2.3.
Server config: Debian Lenny + Php 5.3
Client config: Windows 7 + Firefox 10 (although tested on other configurations)

PS: I see that this new version of the plugin has a pt_BR translation, but it 
has some errors, and seems it was translated from Spanish file. I'll prepare a 
correct pt_BR translation and send to you by email.

Thanks in advance,

Eder Freire
```

Original issue reported on code.google.com by `ederfre...@gmail.com` on 20 Mar 2012 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
Please test http://svn.mail4us.net/plugins/captcha.zip

It is now possible to reload the captcha challenge by Ajax.
```

Original comment by `rol...@liebl.ath.cx` on 21 Mar 2012 at 10:45
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Forgot to mention: Revert you changes in login.html!
```

Original comment by `rol...@liebl.ath.cx` on 21 Mar 2012 at 10:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 21 Mar 2012 at 2:46
- Changed title: **[captcha] Captcha plugin - reload option not working on login page**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 22 Mar 2012 at 10:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks,  friend. The reload feature works perfectly now. You're doing a great 
job with those plugins.

I'd like to suggest one more feature for future versions: the addition of sound 
transcription of the codes, as can be found on reCaptcha and other similar 
services. 

In addition, I can also help you with the translation of plugins to pt_BR when 
necessary.

Thanks and regards,

Eder
```

Original comment by `ederfre...@gmail.com` on 22 Mar 2012 at 4:39

**Comment by GoogleCodeExporter** [comment]
```
I won't add sound support. That's too much overhead. Maybe you want to have a 
look at rcguard plugin:

http://www.roundcubeforum.net/index.php/topic,6233.0.html
```

Original comment by `rol...@liebl.ath.cx` on 23 Mar 2012 at 5:29
